TesseractOCR/G8Tesseract.mm

```diff
@@ -644,6 +645,49 @@ - (NSString *)recognizedHOCRForPageNumber:(int)pageNumber {
     return nil;
 }
 
+- (NSData *)recognizedPDFForImages:(NSArray*)images {
```

Comment:
Another advantage is that the tesseract PDF produces a copyable text in compaarence with Apple API, which only can draw pages.